src/SmartComponents/__tests__/completedTaskDetailsHelpers.tests.js

```diff
@@ -0,0 +1,97 @@
+import {
+  getSelectedSystems,
+  isError,
+  fetchTask,
+  fetchTaskJobs,
+} from '../completedTaskDetailsHelpers';
+import { log4j_task_jobs } from '../CompletedTaskDetails/__tests__/__fixtures__/completedTasksDetails.fixtures';
+import {
+  errorResponse,
+  log4j_task_jobs_systems,
+  successResponse,
+  taskJobsSuccess,
+} from './__fixtures__/completedTaskDetailsHelpers.fixtures';
+import { fetchExecutedTask, fetchExecutedTaskJobs } from '../../../api';
+import * as dispatcher from '../../Utilities/Dispatcher';
+
+jest.mock('../../../api');
+
+describe('getSelectedSystems', () => {
+  it('should return systems', () => {
+    let selectedSystems = getSelectedSystems(log4j_task_jobs);
+    expect(selectedSystems).toEqual(log4j_task_jobs_systems);
+  });
+
+  it('should return no systems', () => {
+    let selectedSystems = getSelectedSystems([]);
+    expect(selectedSystems).toEqual([]);
+  });
+});
+
+describe('isError', () => {
+  it('should return true', () => {
+    expect(isError(errorResponse)).toEqual(true);
+  });
+
+  it('should return false', () => {
+    expect(isError(successResponse)).toEqual(false);
+  });
+});
+
+describe('fetchTask', () => {
+  const setError = jest.fn();
+
+  it('should return fetched data', async () => {
+    fetchExecutedTask.mockImplementation(async () => {
+      return {
+        response: { status: 200 },
```

Comment:
You created and tested these nice `successResponse` and `errorResponse` constants and you're not using them everywhere :smile: It does not save any lines of code, but might avoid a future confusion when a dev edits those constants and changes reflect only somewhere. Maybe also use upper case for constants, but this is super nitpicky.